highfidelity/hifi
# WL 21421 - Script to Fly with Rocket Hands

## Description 
> Create a script that allows users to fly with "Rocket Hands" a la Iron Man. When the script is running, the script should detect when the hands are down at the sides, palms angled downward parallel to the ground, and this should propel the user upward / forward in the direction the user moves.
> 
> The user should probably be propelled in the look direction to prevent confusion and nausea.
> Per worklist https://worklist.net/21421

## Test Plan
1) Download, unzip and run the script attached - 
[rockethands.zip](https://github.com/highfidelity/hifi/files/1120152/rockethands.zip)

2) With your Vive or Oculus Controllers, position your controllers to around your Avatar's hips - this varies per Avatar so you might need to experiment a little.
3) Tilt your controllers so they are parallel to the ground, with a slight tilt at the front downwards.
4) Your Avatar should start flying in the direction you are looking (Make sure you're turning your whole body when looking around)
5) Confirm that you are able to look up directly from the ground and fly straight up.
6) Once you have lifted your controllers, your Avatar should slowly come to a stop whether in mid air or on ground.
